workers/unity/Assets/Playground/Config/ArchetypeConfig.cs

```diff
@@ -13,15 +13,15 @@ public static class ArchetypeConfig
             WorkerTypeToArchetypeNameToComponentTypes = new Dictionary<string, Dictionary<string, ComponentType[]>>
             {
                 {
-                    UnityGameLogic.WorkerType,
+                    nameof(UnityGameLogic),
                     new Dictionary<string, ComponentType[]>()
                     {
                         { CharacterArchetype, new ComponentType[] { typeof(BufferedTransform) } },
                         { CubeArchetype, new ComponentType[] { typeof(BufferedTransform) } }
                     }
                 },
                 {
-                    UnityClient.WorkerType,
+                    nameof(UnityClient),
```

Comment:
Why `nameof`?

Reply:
because nameof provides a string version of the class name which basically represents the worker type. we can have a field storing the worker type, if that's preferred, but I found that this is kind of redundant and more error-prone (e.g. misspelling something)

Reply:
Yeah I'd prefer having a string myself